proposals/NNNN-Deprecated-interfaces-markup.md

```diff
@@ -0,0 +1,133 @@
+# Mark public deprecated methods
+
+* Proposal: [SDL NNN](NNNN-NNNN-Deprecated-interfaces-markup.md)
+* Author: Alexander Kutsan
+* Status: **Awaiting review**
+* Impacted Platforms: Core
+
+## Introduction
+Public methods is part of SDL interface. New features may use some existing interfaces and expect some behavior. 
+Sometimes some features use interfaces that planed to be removed. And after removing interfaces feature can't be released due to conflicts in code.
+
+## Motivation
+Marking method as deprecated should trigger a compile time warning, but should not block project assemble.
+Reverse feature support plugins that can be created by 3rd party developers.
+Each public method should work to handle backward capabilities with other developers plugins code.
+Also some classes also can be deprecated and creating instances of this classes should cause compile time warnings also.
```

Comment:
Please update to "Also some classes can be deprecated and creating instances of these classes should cause compile time warnings."